### ISSUE TITLE
NR Officium Defunctorum : neume transposition errors?

### ISSUE THREAD

**Issue by Suinesra** [Nocturnale-Romanum/nocturnale-romanum#49]
Sandhofe: Nocturnale Romanum
Officium Defunctorum
Antiphon: _Complaceat tibi_, pages [195] & [196]

The melody of this antiphon is used elsewhere in the Gregorian repertoire (although at the moment I can't recall where). 
The cadence at the end of the arsis (over _ut eripias me;_) in other usages ends on Do, not on Re.

The same correction ought to be made in the antiphon repeated at the end of the psalm (page [196]).

[NR [195].pdf](https://github.com/user-attachments/files/17750941/NR.195.pdf)

**Comment by xmarteo** [comment]
You are right, and critical analysis of the sources is in favor of the Do.

Fixed in https://nocturnale.marteo.fr/chant/ODEFN3A1/

Sandhofe correctly uses the same formula in https://nocturnale.marteo.fr/chant/F3N3A2/ (p. 44 in his NR)
